README.md

```diff
@@ -1 +1,39 @@
-# To-Do-List-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do-List-App
+
+> This is a simple page that lists the things that you need to do and allows you to mark them as complete when you're finished.
+
+## Live Demo
+> Live demo coming soon ⏳
+
+## Built With
+
+- HTML
+- CSS 
+- Javascript
+
+### Technologies used 
+- Git, linters and Github  
+
+## Author
+
+👤 **Abel Gebeyehu**
+
+- GitHub: [@AbelG101](https://github.com/AbelG101)
+- LinkedIn: [Abel Gebeyehu](https://www.linkedin.com/in/abel-gebeyehu-779743183/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] - In a bid to professionally document your project, you have only included the technologies used. Assuming that the person to use this project is a complete noob, he or she will not know how to run your project. Kindly include steps on how one is supposed to run your project starting from how to install all the packages and dependencies.
- [ ] Also, I have realized that your MIT.md file link is broken. Kindly include it as well.